src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1072,11 +1072,13 @@ private Task<BatchAccount> CreateBatchAccountAsync()
 
         private Task<IResourceGroup> CreateResourceGroupAsync()
         {
+            var tags = DelimitedTextToDictionary(configuration.Tags.Trim());
             return Execute(
```

Comment:
Add blank line between these two statements - rule is that multi line block needs to be separated from other code by a blank line.